src/test/playwright/e2e/exam/ExamAssessment.spec.ts

```diff
@@ -381,7 +381,7 @@ async function startAssessing(
     exerciseAssessment: ExerciseAssessmentDashboardPage,
 ) {
     await examManagement.openAssessmentDashboard(courseID, examID, timeout);
-    await courseAssessment.clickExerciseDashboardButton();
+    await courseAssessment.clickExerciseDashboardButton(0);
```

Comment:
```suggestion
    await courseAssessment.clickExerciseDashboardButton();
```
The default is already 0. Or remove the default if you don't want to have one